library/src/androidTest/java/com/etrusted/android/trustbadge/library/model/TrustbadgeDataBadResponseAndroidTest.kt

```diff
@@ -0,0 +1,51 @@
+/*
+ * Created by Ali Kabiri on 30.1.2023.
+ * Copyright (c) 2023 Trusted Shops GmbH
+ *
+ * MIT License
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.etrusted.android.trustbadge.library.model
+
+import com.etrusted.android.trustbadge.library.common.internal.ServerResponses
+import com.google.common.truth.ExpectFailure.expectFailure
+import com.google.common.truth.Truth.assertThat
+import org.json.JSONException
+import org.junit.Assert.assertThrows
+import org.junit.Test
+//import kotlin.test.assertFailsWith
+
```

Comment:
Is it ok to remove commented line of code?

Reply:
nice catch! Yes, I'll remove it